app/components/dashboard/hypertension/overdue_patients_called_component.rb

```diff
@@ -28,11 +28,11 @@ def graph_data
     {
       overduePatients: data[:overdue_patients],
       overduePatientsCalled: data[:patients_called],
-      overduePatientsCalledRate: data[:patients_called_rates],
+      overduePatientsCalledRate: data[:patients_called_rates].map { |k, v| [k, cap_percentage(v)] }.to_h,
       startDate: @period.advance(months: -17),
-      calledWithResultAgreedToVisit: data[:patients_called_with_result_agreed_to_visit_rates],
-      calledWithResultRemindToCallLater: data[:patients_called_with_result_remind_to_call_later_rates],
-      calledWithResultRemoveFromOverdueList: data[:patients_called_with_result_removed_from_list_rates],
+      calledWithResultAgreedToVisit: cap_call_result_rates(:patients_called_with_result_agreed_to_visit_rates),
+      calledWithResultRemindToCallLater: cap_call_result_rates(:patients_called_with_result_remind_to_call_later_rates),
+      calledWithResultRemoveFromOverdueList: cap_call_result_rates(:patients_called_with_result_removed_from_list_rates),
       chartProportionalPercentageCalledWithResultAgreedToVisit: proportional_call_rate(:patients_called_with_result_agreed_to_visit_rates),
```

Comment:
overduePatientsCalledRate needs to be used to calculate the proportion.

overduePatientsCalledRate / 100 * patients_called_with_result_agreed_to_visit_rates

examples:
below 100% called
80% of patients called
40% patients agree to visit
20% remind to call
40% remove
80 / 100 * 40 = 32 -> 32% is the area (height of the band) for agree to visit
80 / 100 * 20 = 16 -> 16% is the area (height of the band) for remind
80 / 100 * 40 = 32 -> 32% is the area (height of the band) for removed
(16% + 32% + 32% = 80%)

over 100% called
140% of patients called
we cap 140% of patients called to 100%
40% patients agree to visit
20% remind to call
40% remove
100 / 100 * 40 = 40 -> 40% is the area (height of the band) for agree to visit
100 / 100 * 20 = 20 -> 20% is the area (height of the band) for remind
100 / 100 * 40 = 40 -> 40% is the area (height of the band) for removed
(40% + 20% + 40% = 100%)